packages/web/src/common/store/cache/collections/commonSagas.ts

```diff
@@ -481,6 +482,12 @@ function* publishPlaylistAsync(
     playlist,
     action.dismissToastKey
   )
+
+  for (const track of playlist.tracks ?? []) {
```

Comment:
anything we need to do here with error handling? also what is the consideration for users notifcations? seems like they would get "artist released new album" and then "artist released n tracks"?. lets confirm behavior here.... might have to be a backend thing, or we pass a "no notif" flag or something?

Reply:
Yeah good call on the notifs. Error handling we can think about. I could see doing a few things:
- don't publish tracks if the album fails
- don't publish album if track(s) fail
- best effort everything but notify the user if one or more tracks didn't publish

Reply:
as a first pass unlock this is great though, and we can chat as a group tomorrow?

Reply:
reed and I talked and decided to move the cascading track publish to the success callback for collection publish

adding ticket to explore error edit handling in general, and also ticket for the cascading notifications issue